packages/richtext/src/utils/parse-tiptap-json.ts

```diff
@@ -0,0 +1,22 @@
+import { generateHTML } from '@tiptap/html';
+import { JSONContent, Extensions } from '@tiptap/core';
+import { debug } from '@sitecore-content-sdk/core';
+import StarterKit from '@tiptap/starter-kit';
+
+export const parseTiptapJSON = (content: JSONContent[], extensions: Extensions = []) => {
+  // prevent warns when StarterKit is passed in extensions
+  const nonDuplicateExtensions = Array.from(new Set([StarterKit, ...extensions]));
+  let result = '';
+  try {
+    result = generateHTML(
+      {
```

Comment:
AFAIK `type: doc` is part of field content, so you don't need to provide it, e.g. here is the example of rich text field in CS:
![image](https://github.com/user-attachments/assets/21f9bb21-5424-45c1-9d22-3675037ca25f)